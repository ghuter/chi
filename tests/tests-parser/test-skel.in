A :: skeleton Module {
	f :: fun();
}
