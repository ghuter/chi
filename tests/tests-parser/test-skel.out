SMODSKEL(A, SSIGN(f) : void}) : Module
